src/RadhydroSimulation.hpp

```diff
@@ -123,6 +123,8 @@ template <typename problem_t> class RadhydroSimulation : public AMRSimulation<pr
 
 	amrex::Long radiationCellUpdates_ = 0; // total number of radiation cell-updates
 
+	amrex::Real Gconst_ = 1.0; // gravitational constant G (code units)
+
```

Comment:
Is this where it should be defined, if it's only used in StarCluster problem?

Reply:
We should probably have a `Poisson_Traits` class or something like that and define it there.

Reply:
Actually, you could just make it a runtime parameter.

Reply:
But then we'd have to include it in every test problem?

Reply:
We could set it to the CGS value by default. Then, it would only have to be included if the problem didn't use CGS units, and also used Poisson gravity.

Reply:
Where should the default value be set? In here?

```
        explicit RadhydroSimulation(amrex::Vector<amrex::BCRec> &BCs_cc, amrex::Vector<amrex::BCRec> &BCs_f$
        {
               	defineComponentNames();
               	// read in runtime parameters
                readParmParse();
               	// set gamma
               	amrex::ParmParse eos("eos");
               	eos.add("eos_gamma", quokka::EOS_Traits<problem_t>::gamma);

               	// initialize Microphysics params
                init_extern_parameters();
                // initialize Microphysics EOS
                amrex::Real small_temp = 1e-10;
                amrex::Real small_dens = 1e-100;
                eos_init(small_temp, small_dens);
        }
```

Reply:
It can be set as a member variable in the definition of the `RadhydroSimulation` class.

Reply:
Okay, I made it a runtime param that is then read in parmParse. But, I set the default value to CGS, so problems that dont define it should use the default value. Do the changes make sense @BenWibking

Reply:
Yeah, I think your changes make sense 👍